keras_cv/datasets/waymo/transformer.py

```diff
@@ -481,6 +482,26 @@ def _get_point_label(
     )
 
 
+def _inv_loc(rot: tf.Tensor, loc: tf.Tensor) -> tf.Tensor:
```

Comment:
I think this function is already provided in voxel_utils.py, can you re-use that one?

Reply:
Seems no functions for invert location? I can see get_yaw_rotation but not fit here.

Reply:
Oops, sorry, should be here:
https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/object_detection3d/center_net_label_encoder.py#L55
Can we extract this out into the voxel_utils.py?

Reply:
Sure, also move internal utils to voxel_utils.